src/actions-types/index.js

```diff
@@ -0,0 +1,4 @@
+import * as passwordAction from './PasswordActionsType';
+import * as notificationActionTypes from './notificationActionTypes';
+
+export { passwordAction, notificationActionTypes };
```

Comment:
@luc-tuyishime, to be constant in the codebase let all of us use the same style and convention, as you know that we have a folder for actions, it would be better if you export all the actions types prefixed whit `type`
```suggestion
export { passwordActionsTypes, notificationActionTypes };
```

Reply:
Yes yes you are right, i have already applied the changes